### ISSUE TITLE
Text Highlighting

### ISSUE THREAD

**Issue by BruceSaltzman** [believer/roam-night-owl#1]
Beautiful theme. Text highlighting is bright yellow which hides the text underneath. Looked through CSS to find reference to block highlighting, which I found, however was unable to change. Thanks.

**Comment by believer** [comment]
Thank you! Great catch, I haven't really used highlighting so I missed that case. I'll get a fix out today.

**Comment by BruceSaltzman** [comment]
Thanks very much.

**Comment by BruceSaltzman** [comment]
Thanks for posting the update. Highlighting works perfectly now.

If you don't mind, I found one other thing that might interest you. I have a table I refer to frequently. The table is wider than the main container. In other themes I've tried, a bottom (horizontal) scrollbar is provided to enable viewing of the hidden parts. I can't seem to locate the scrollbar in your theme. Just wondering if it's there and I can't see it because of the dark background.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐
